etna/Makefile

```diff
@@ -7,10 +7,12 @@ export BUILD_REQS:=../docker/etna-base $(BUILD_REQS)
 test::
 	@ docker-compose run --rm etna_app bundle exec rspec
 	@ docker-compose run -e RUN_NPM_INSTALL=1 -e SKIP_RUBY_SETUP=1 --rm etna_app npm test
+	# We will have to add in a command here to test the R stuff, too.
 
 bash::
 	@ docker-compose run -e SKIP_RUBY_SETUP=1 --rm etna_app bash
 
 release-test::
 	docker run --rm -e APP_NAME=etna -e RELEASE_TEST=1 -e SKIP_DB_WAIT=1 -e CI_SECRET=$${CI_SECRET} -e IS_CI=$${IS_CI} --network monoetna_default $(fullTag) /entrypoints/development.sh bundle exec rspec
 	docker run --rm -e APP_NAME=etna -e RELEASE_TEST=1 -e SKIP_RUBY_SETUP=1 -e RUN_NPM_INSTALL=1 -e SKIP_DB_WAIT=1 --network monoetna_default $(fullTag) /entrypoints/development.sh npm test
+	# We will have to add in a command here to test the R stuff, too.
```

Comment:
Seems like this PR altogether adds R installation into the docker image & preps for having R tests via GitHub actions, both of which are great.  I'm just commenting for my own understanding...

Would the stub test lines here be replaced with just like a line of R code that will work only if R is installed?  Or would the goal be to point to the magmaR test suite?

Reply:
Yup, exactly! These commented out lines would basically be replaced with a shell command (or however R tests are invoked?) that would run the actual magmaR test suite -- the idea is that on every commit, GitHub will automatically run these set of tests (Ruby tests, JavaScript tests, and R tests) and notify us of any failures. Each PR should add or modify tests as appropriate, to make sure that things don't break. And we don't merge code unless all tests pass and you see the green checkmark on the PR.